letsencrypt/storage.py

```diff
@@ -275,11 +289,13 @@ def current_version(self, kind):
         pattern = re.compile(r"^{0}([0-9]+)\.pem$".format(kind))
         target = self.current_target(kind)
         if target is None or not os.path.exists(target):
+            logger.debug("File does not exist")
```

Comment:
You should make these log messages maximally meaningful.  Something like this:
`logger.debug("Current-version target for %s does not exist at %s", kind, target)`